testing/configs/config_artifactdownloader_output.yaml

```diff
@@ -6,3 +6,6 @@ file_groups:
     path: testing/resources/artifactdownloader_output
     artifact: perfherder-data
     run_number: 1
+
+analysis:
```

Comment:
Please remove the analysis section from this config and make a new config if you need it.